connector_search_engine/components/mapper.py

```diff
@@ -20,8 +19,16 @@ def __init__(self, work):
         super(JsonExportMapper, self).__init__(work)
         exporter = work.index.exporter_id
         self._json_parser = exporter.get_json_parser()
-        if _f("id") not in self._json_parser["fields"]:
-            self._json_parser["fields"].append(_f("id"))
+        mapping_id = [
```

Comment:
I'd use `_record_id_key`

Reply:
For that case no.
The idea is to always export the key "id" (so we use this key in the template for linking product, categ, brand ....)
then only for Algolia case we set the key "objectID" with the value of the "id" so in the template you still continue to use the key "id" and never never never use the key "objectID".

It will make all frontend code more compatible